src/index.html

```diff
@@ -35,6 +35,14 @@
         </div>
       </div>
     </div>
+
+    <div class='footer-container'>
+      <form class='footer-form'>
+        <p class='footer-info'>Get info on new updates</p>
+        <input class='footer-text-field' type='text'>
+        <input class='footer-submit' type='submit' value='Subscribe'>
```

Comment:
although this is a valid way of doing a submit button a better way is by using a button with type submit. this allows you to add any content inside the button while using the input type submit the content will be limited to what the value attribute supports (text).